examples/learning/helloworld64.html

```diff
@@ -40,36 +40,35 @@ <h1>'Hello, world!' example</h1>
   // Opening PDF by passing its binary data as a string. It is still preferable
   // to use Uint8Array, but string or array-like structure will work too.
   var loadingTask = pdfjsLib.getDocument({ data: pdfData, });
-  loadingTask.promise.then(function(pdf) {
+  (async function () {
```

Comment:
top level awaits are available since Firefox 89. Should this syntax be preferred ?